tables/tests/test_basics.py

```diff
@@ -2421,10 +2421,9 @@ def test_str_names(self):
         # see gh-42
         d = {'name': tables.Int16Col()}
         descr = Description(d)
-        self.assertEqual(sorted(descr._v_names), sorted(d.keys()))
+        self.assertEqual(sorted(descr._v_names), sorted(d))
```

Comment:
Maybe in this specific case using ``.keys()`` is more easy to understand for the reader.

Reply:
Okay, I will revert that.